### PR TITLE
Add isLiked field to post data

### PR DESCRIPTION
**Pull request by picklechips**
`isLiked` was never being sent when toggling a like so it was always undefined, resulting in likes not working.

Fixes https://github.com/CGUC/skybunk-mobile/issues/193

### CODE REVIEW

File: components/Post/Post.jsx [hunk 1]
```diff
@@ -148,6 +148,7 @@ export default class Post extends React.Component {
       data.likes--;
       data.usersLiked = _.filter(data.usersLiked, user => user._id !== loggedInUser._id);
     } else {
+      data.isLiked = true;
```
Comment (aopal) [inline components/Post/Post.jsx]: Should probably also add `data.isLiked = false;` under `if (data.likes < 0)` below (that should probably be a <= too)

**Comment by scholvat** [comment]
Not the way I intended this code to work, but it would fix it (assuming Nuj's comment is resolved). I intended to get rid of the isLiked variable and just reference the usersLiked field, since I wanted to just keep one variable that stores all the information for lies. The part I forgot to update is like 109 of Comments.jsx and line 136 of Feed.jsx, where it references data.isLiked instead of data.usersLiked.includes(loggedInUser._id)

**Comment by picklechips** [comment]
> Not the way I intended this code to work, but it would fix it (assuming Nuj's comment is resolved). I intended to get rid of the isLiked variable and just reference the usersLiked field, since I wanted to just keep one variable that stores all the information for likes. The part I forgot to update is like 109 of Comments.jsx and line 136 of Feed.jsx, where it references data.isLiked instead of data.usersLiked.includes(loggedInUser._id)

Gotcha, I just noticed in Feed.jsx we were relying on `isLiked` even though it was always undefined. 
I'll update it as you described since that seems a little cleaner to me. (Although may be inefficient for hundreds of thousands of likes, but that's irrelevant.)

**Comment by picklechips** [comment]
Ping @aopal @scholvat

Updated

**Pull request merged**